go-controller/pkg/node/node.go

```diff
@@ -424,6 +433,11 @@ func (n *OvnNode) Start(ctx context.Context, wg *sync.WaitGroup) error {
 		}
 	}
 
+	if config.OvnKubeNode.Mode != types.NodeModeDPUHost {
```

Comment:
Why do we need to do this only if mode is not `types.NodeModeDPUHost`?  Can't we just do it all the time?

Anyway, we could just move this code in the `else` branch above.